.github/workflows/test.yml

```diff
@@ -59,10 +59,12 @@ jobs:
           path: .venv
           key: venv-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
 
+      # TODO: remove pip install after optuna fix
```

Comment:
Am I write that user should also install the correct version of sqlalchemy? Should't we inform him somehow about it?

Reply:
If he has incorrect version he will face some errors working with databases in optuna (it is optuna's problem, not ours).